### PR TITLE
[READY] Made Biogens easily accessible by Cooks on Delta and Box

### PR DESCRIPTION
**Pull request by CinderWC**
## About The Pull Request
This PR makes it so that the Cook has access to the Biogen. This is done by putting the Biogenerator into a wall between Botany and the Kitchen.

## Why It's Good For The Game
Kilo, Pubby and Meta have it this way. So why shouldn't Box and Delta?

## Changelog
:cl:
tweak: Moves spare delta bio-gene to puplic access
tweak: Moves Box bio-gene to cook-wall for easy access
tweak: Tweaked Botany and Kitchen on both Box and Delta to accomodate the Biogen changes
/:cl:

### CODE REVIEW

**Comment by jjpark-kb** [comment]
I recommend instead of a table, use a chem dispenser, specifically this one: https://github.com/Skyrat-SS13/Skyrat13/blob/a06cee73bea4b879132415fbd7591a8b1a98d7b7/code/modules/reagents/chemistry/machinery/chem_dispenser.dm#L609.
However, there is a problem with that one, it has fully upgraded parts. So perhaps make a modular version that is similar in nature, but doesnt have upgraded parts, and doesnt have cryo.

**Comment by CinderWC** [comment]
@jjpark-kb Tried making a modular chem dispenser. Im too stupid to do it so im not going to.

File: _maps/map_files/BoxStation/BoxStation_Skyrat.dmm [hunk 1]
```diff
@@ -20476,7 +20488,7 @@
 /area/storage/tools)
 "aTM" = (
 /obj/structure/table,
-/obj/item/reagent_containers/food/snacks/mint,
```
Comment (Akrilla) [inline _maps/map_files/BoxStation/BoxStation_Skyrat.dmm:20479]: Removing the only mint

Comment (CinderWC) [reply _maps/map_files/BoxStation/BoxStation_Skyrat.dmm:20479]: Is the mint really that important?

Comment (Akrilla) [reply _maps/map_files/BoxStation/BoxStation_Skyrat.dmm:20479]: I mean I don't know why it was removed

Comment (CinderWC) [reply _maps/map_files/BoxStation/BoxStation_Skyrat.dmm:20479]: It was removed to move the scrambler thingy that grinds things over

Comment (Akrilla) [reply _maps/map_files/BoxStation/BoxStation_Skyrat.dmm:20479]: Please readd it back somewhere else

Comment (CinderWC) [reply _maps/map_files/BoxStation/BoxStation_Skyrat.dmm:20479]: I mean i find this pretty petty. But sure.

File: _maps/map_files/BoxStation/BoxStation_Skyrat.dmm [hunk 2]
```diff
@@ -61214,9 +61214,7 @@
 	id = "kitchen";
 	name = "kitchen shutters"
 	},
-/obj/item/reagent_containers/food/snacks/cheesynachos{
```
Comment (CinderWC) [inline _maps/map_files/BoxStation/BoxStation_Skyrat.dmm]: PepeHands

**Comment by davidchan13** [comment]
Hey someone made the thing I've been doing for years a map default.
Though not sure what you mean about meta's biogen starting chef accessible, it starts by the DNA machine in the bottom right of botany on meta.

**Comment by CinderWC** [comment]
> Though not sure what you mean about meta's biogen starting chef accessible, it starts by the DNA machine in the bottom right of botany on meta.

Meta has not had it's Biogenerator position modified as the Cook already has access to Botany by default.
It has, however received a new botany chem dispenser that is on every new map

**Pull request merged**